workshop/exercise-7/README.md

```diff
@@ -133,4 +89,34 @@ Application Metrics 5.0.3.201908230949 (Agent Core 4.0.3)
 
 Test it out with `curl` and you'll see the helmet headers.
 
-**Congratulations!!** We've got our Kabanero instance updated to use our custom collection.
+Use the `appsody stop` command in another terminal in the same directory to stop the running application.
+
+### 3. Prepare the new application for deployment
+
+Before you can deploy this application using the pipeline you will be building in the upcoming exercises, there needs to be a deployment file added to the application. The deployment file is called `app-deploy.yaml`. This file was created automatically for you in day 1 exercise 3 with the `appsody deploy` command.
+
+It can also be created without deploying an application using the `--generate-only` flag. Run this command:
+
+```bash
+appsody deploy --generate-only
+```
+
```

Comment:
add a line between these code blocks, like "You should see output like the following..."